### ISSUE TITLE
Discord Server - Trasparency Upgrade

### ISSUE THREAD

**Issue by StoneMcYT** [Avdan-OS/roadmap#1]

**Comment by StoneMcYT** [comment]
Finished. (This was all copy pasted from a discord channel were the setting was casual conversation) 
TLDR
- Making move to make all dev & designer chats open 
Dev chanels become view only to public that claims a specfic role in #infomation  - docsite and website is read + chat
Designer perms become read only for the public (requires a role claim) and designers can create threads if they want peoples opinion
a #submissions dedicated for peoplel who want to submit their designs

Notion and what not becomes public

**Comment by StoneMcYT** [comment]
Finished. (This was all copy pasted from a discord channel were the setting was casual conversation) 
TLDR
- Making move to make all dev & designer chats open 
Dev chanels become view only to public that claims a specfic role in #infomation  - docsite and website is read + chat
Designer perms become read only for the public (requires a role claim) and designers can create threads if they want peoples opinion
a #submissions dedicated for peoplel who want to submit their designs

Notion and what not becomes public
